Search/SearchRoutes.js

```diff
@@ -0,0 +1,204 @@
+import * as stravaDao from "../Strava/strava-dao.js";
+import * as stravaRoutes from "../Strava/StravaRoutes.js";
+import axios from "axios";
+
+const ONE_DAY_SECONDS = 24 * 60 * 60;
+const activities_link = "https://www.strava.com/api/v3/athlete/activities";
+
+export default function SearchRoutes(app) {
+    
+    // For testing, delete later
+    const sampleResponse = 
```

Comment:
can we delete this now or do you still need it?

Reply:
Yeah we still need it, I removed the comment